source/reference/minio-server/minio-server.rst

```diff
@@ -277,12 +277,23 @@ Core Configuration
 .. envvar:: MINIO_CONFIG_ENV_FILE
 
    Specifies the full path to the file the MinIO server process uses for loading environment variables.
+   
+   For ``systemd``-managed file, setting this value to the environment file allows MinIO to reload changes to that file on using :mc-cmd:`mc admin service restart` to restart the deployment.
```

Comment:
```suggestion
   For ``systemd``-managed files, setting this value to the environment file allows MinIO to reload changes to that file on using :mc-cmd:`mc admin service restart` to restart the deployment.
```

Not sure if this might be `For systemd-managed files` or `For a systemd-managed file`, but here's a suggestion for the first one.